Extension/src/LanguageServer/client.ts

```diff
@@ -1392,6 +1400,8 @@ export class DefaultClient implements Client {
         if (document.uri.scheme === "file") {
             openFileVersions.set(document.uri.toString(), document.version);
         }
+        // Log warm start.
+        clientCollection.timeTelemetryCollector.setActivationTime(document.uri.fsPath);
```

Comment:
`.fsPath` is not the URI.  We should use URIs everywhere to avoid messing with the capitalization.  I believe it should be just `.path` here.